library/persistence/storage.py

```diff
@@ -155,3 +156,10 @@ def delete_invoice(invoice: Invoice):
         index = invoices.index(invoice)
         if index >= 0:
             invoices.remove(invoice)
+
+    @staticmethod
+    def find_book(self,book_copy: "BookCopy") -> "Book":
```

Comment:
```suggestion
    def find_book(self,book_copy: "BookCopy") -> "Book":
```
Not sure if you can pass self as input argument to static method. Maybe Maintain an Inventory class, and pass as input arg